lesson_1/task_5/DifferentNumbers.java

```diff
@@ -0,0 +1,24 @@
+public class DifferentNumbers {
+    public static void main(String[] args) {
+        int n = 2516;
+        String s = String.valueOf(n);
+        int count = 0;
+        if (s.length() == 4) {
+            for (int i = 0; i < s.length(); i++) {
+                for (int j = 0; j < s.length(); j++) {
+                    if (s.charAt(i) == s.charAt(j) && j != i){
```

Comment:
if (...) {  You shoud use space between ) {

Reply:
was changed!)

Reply:
you should just use hotkey "Reformat Code"

Reply:
Thank you for the hotkey!!